CMakeLists.txt

```diff
@@ -198,6 +201,7 @@ endif() # WIN32
 
 find_package(LZ4)
 find_package(ZSTD)
+find_package(PNG)
```

Comment:
Is this used for anything?

Reply:
Good catch.  This was for using libPNG which I dropped.